lib/handlers/allow.js

```diff
@@ -68,6 +69,21 @@ function allow (mode) {
         }
       }
     }
+
+    // check user is owner. Find owner from /meta
+    let graph
+    try {
+      // do not check for permission. Owner has all rights.
```

Comment:
Just  a heads-up that the owner has implicit Control privilege on all resources in the Storage, not necessarily Read-Write on the resource itself.

Reply:
Agree, only control, not necessarily read or write.

Reply:
So something like
```
    if (mode === 'Control') {}
```
around this bit seems it would do that

Reply:
was resolved with a check that resource must be an acl.
`mode = Control` for acl's is done in `acl-checker.js`